### DISCUSSION TITLE
Pre-deloy linter?

### DISCUSSION THREAD

**Discussion by Dutch-TS-Dev** [vercel/community#5632] [Help] [answered]
### Summary

Hey everybody, everytime I deploy my Vercel app I get tons of warnings. I am extremely satisfied with Vercel except forthis part. Im working on a small mvp, and my deploys keep getting rejected.

Is there a way to completely disable vercel linter? or at least run a pre-deploy linter locally so I dont have to wait every time before I find out my deploy has failed?

### Example

text speaks for itself

### Steps to Reproduce

Deploy to vercel

**Comment by TooTallNate** [comment, marked as answer]
You can run `vercel build` locally.
